lib/prx_auth/rails/ext/controller.rb

```diff
@@ -34,10 +35,36 @@ def authenticate!
         redirect_to PrxAuth::Rails::Engine.routes.url_helpers.new_sessions_path
       end
 
+      def prx_auth_needs_refresh?(jwt_ttl)
+        request.get? && jwt_ttl < PRX_JWT_REFRESH_TTL
+      end
+
       def current_user
         prx_auth_token
       end
 
+      def current_user_info
+        session[PRX_USER_INFO_SESSION_KEY] ||= fetch_userinfo
```

Comment:
As I was writing this, I started thinking ... we are really depending on non-cookie session storage now.  The JWT alone is going to overfill it.  I wonder if that should/could be built into the gemspec dependencies?